browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -11623,15 +12009,29 @@ index 0000000000000000000000000000000000000000..3f7bfd524cff408bfaea96177e39a4e2
 +#if PLATFORM(MAC)
 +void InspectorScreencastAgent::encodeFrame()
 +{
++    if (!m_encoder && !m_screencast)
++        return;
++    RetainPtr<CGImageRef> imageRef = m_page.pageClient().takeSnapshotForAutomation();
++    if (m_screencast && m_screencastFramesInFlight <= 1) {
++        auto cfData = adoptCF(CFDataCreateMutable(kCFAllocatorDefault, 0));
++        WebCore::encodeImage(imageRef.get(), CFSTR("public.jpeg"), 0.5, cfData.get());
```

Comment:
- 0.5 -> m_screencastQuality?
- What is this public.jpeg?

Reply:
- Done
- Don't know